apps/web/lib/team/[slug]/getServerSideProps.tsx

```diff
@@ -31,10 +32,12 @@ const getTheLastArrayElement = (value: ReadonlyArray<string> | string | undefine
 export const getServerSideProps = async (context: GetServerSidePropsContext) => {
   const slug = getTheLastArrayElement(context.query.slug) ?? getTheLastArrayElement(context.query.orgSlug);
 
-  const { isValidOrgDomain, currentOrgDomain } = orgDomainConfig(
-    context.req,
-    context.params?.orgSlug ?? context.query?.orgSlug
-  );
+  const redirectDomainSlug = context.query?.redirectDomainSlug;
```

Comment:
I don't think we need this parameter with the new approach. The only reason I think it could be useful is to just inform acme.com about the source of traffic which I believe can be determined from referrer.

Reply:
we still need this as i mentioned earlier, if isValidOrgDomain = false  currentOrgDomain=null, this results to notfound page after redirect

Reply:
Yeah that is true